spec/controllers/idv/gpo_verify_controller_spec.rb

```diff
@@ -163,6 +173,69 @@
           action
         end
       end
+
+      context 'threatmetrix enabled' do
+        let(:threatmetrix_enabled) { true }
+
+        context 'with threatmetrix status of "reject"' do
+          let(:proofing_components) do
+            ProofingComponent.create(
+              user: user, threatmetrix: true,
+              threatmetrix_review_status: 'reject'
+            )
+          end
+
+          it 'redirects to the sad face screen' do
+            expect(@analytics).to receive(:track_event).with(
+              'IdV: GPO verification submitted',
+              success: true,
+              errors: {},
+              pending_in_person_enrollment: false,
+              threatmetrix_check_failed: true,
+              enqueued_at: user.pending_profile.gpo_confirmation_codes.last.code_sent_at,
+              pii_like_keypaths: [[:errors, :otp], [:error_details, :otp]],
+            )
+
+            action
+
+            expect(response).to redirect_to(idv_setup_errors_url)
+          end
+
+          it 'does not show a flash message' do
+            expect(flash[:success]).to be_nil
+            action
+          end
+
+          it 'does not dispatch account verified alert' do
+            expect(UserAlerts::AlertUserAboutAccountVerified).not_to receive(:call)
+            action
+          end
+        end
```

Comment:
Relevant Slack thread discussing this: https://gsa-tts.slack.com/archives/CNCGEHG1G/p1671730304506409